frontend/src/components/Elections.tsx

```diff
@@ -79,15 +89,16 @@ const Elections = ({ authSession }) => {
                                     <TableCell > {election.start_time ? new Date(election.start_time).toLocaleString() : ''}</TableCell>
                                     <TableCell >{election.end_time ? new Date(election.end_time).toLocaleString() : ''}</TableCell>
                                     <TableCell >{limit(election.description, 30) || ''}</TableCell>
-                                    <TableCell ><Button variant='outlined' href={`/Election/${String(election.election_id)}${authSession.isLoggedIn()? '/admin' : ''}`} > View </Button></TableCell>
+                                    <TableCell ><Button variant='outlined' href={`/Election/${String(election.election_id)}${authSession.isLoggedIn() ? '/admin' : ''}`} > View </Button></TableCell>
```

Comment:
I don't think the archival button needs to be on the elections page. It makes sense for us since we're frequently bulk creating and archiving elections, but not for the common user

Reply:
Fair, I really only added it so I could declutter my test elections faster.

Reply:
Done